parsers/CA_SK.py

```diff
@@ -115,7 +115,9 @@ def fetch_consumption(
     # Set the headers to mimic a user browser as the API will return a 403 if not.
     headers = {"user-agent": USER_AGENT}
 
-    response: Response = session.get(CONSUMPTION_URL)  # , headers=headers)
+    response: Response = session.get(
+        CONSUMPTION_URL, verify=False
+    )  # , headers=headers)
```

Comment:
It's cleaner if you remove the dummy comment

```suggestion
    response: Response = session.get(CONSUMPTION_URL, verify=False)
```

Reply:
done